runtime/doc/eval.txt

```diff
@@ -7380,6 +7380,8 @@ repeat({expr}, {count})					*repeat()*
 resolve({filename})					*resolve()* *E655*
 		On MS-Windows, when {filename} is a shortcut (a .lnk file),
 		returns the path the shortcut points to in a simplified form.
+		When {filename} is a symbolic link or junction point, return
+		the path to the target.
```

Comment:
If this function always returns a full path to a reparse point, this should be s/path/full path/, I think.